### PR TITLE
Fix Exposure of Sensitive Information

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Resolves:
https://bugzilla.redhat.com/show_bug.cgi?id=2085342

Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.2.0",
+    "eventsource": "^1.1.1",
```
Comment (GowthamShanmugam) [inline package.json:87]: in BZ it is mentioned all builds which are < 2.0.2 has this issue, I think you need to use the 2.0.2 build: https://github.com/EventSource/eventsource/releases/tag/v2.0.2. Please confirm with the security team once.

Comment (TimothyAsirJeyasing) [reply package.json:87]: Will check with 2.0.2. However 1.1.1 is also fix this issue is mentioned in https://huntr.dev/bounties/dc9e467f-be5d-4945-867d-1044d27e9b8e at the end of conversation

**Review by bipuladh — APPROVED** [review]

**Pull request merged**